backend/layers/business/business.py

```diff
@@ -340,6 +413,27 @@ def update_collection_version(
             self.database_provider.save_collection_publisher_metadata(version_id, publisher_metadata_to_set)
         self.database_provider.save_collection_metadata(version_id, new_metadata)
 
+        if all(
+            [not ignore_doi_update, new_doi != current_doi, FeatureFlagService.is_enabled(FeatureFlagValues.SCHEMA_4)]
```

Comment:
`not ignore_doi_updated` would be easier to understand if positive logic is used. ex: `updated_doi`